hack/generator/pkg/config/type_filter.go

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package config
+
+// TypeFilterAction defines the possible actions that should happen for types matching the filter
+type TypeFilterAction string
+
+const (
+	// TypeFilterActionIncludeType indicates that any type matched by the filter should be included in the type graph
+	TypeFilterActionIncludeType TypeFilterAction = "include"
```

Comment:
And then these would be `TypeFilterInclude` and `TypeFilterPrune`, right?

Reply:
Yep! much cleaner